### PR TITLE
SimplifyVCF: Add option to declare all unfiltered variants as passing/failing.

### PR DESCRIPTION
**Pull request by tmooney**

### CODE REVIEW

File: lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm [hunk 1]
```diff
@@ -28,6 +28,13 @@ class Genome::Model::SomaticVariation::Command::SimplifyVcf {
             doc => 'If set to true, simplify the indel file as well and merge the indels with snvs',
         },
     ],
+    has_optional_input => [
+        unfiltered_status => {
+            is => 'Text',
+            valid_values => ['pass', 'fail', undef],
+            doc => 'If set, any unfiltered variants will be considered to have passed or failed as specified. Otherwise, this tool will refused to process unfiltered variants.',
+        },
```
Comment (ebelter) [inline lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm]: 'will refused' should be 'will refuse'.

File: lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm [hunk 2]
```diff
@@ -284,16 +291,23 @@ sub simplify_vcf{
...
-    my $tumor_filter_value = $tumor[$ft_pos];
+
     $variant_count++;
-    unless ($tumor_filter_value){
-      print Dumper @line;
+    my $tumor_filter_value;
+    if (defined $ft_pos) {
+        $tumor_filter_value = $tumor[$ft_pos];
+        unless ($tumor_filter_value){
+            print Dumper @line;
```
Comment (ebelter) [inline lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm]: Not part of this PR, but dumping @line lead to anything productive?

File: lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm (hunk 1 above)
Comment (tmooney) [inline lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm]: Indeed--updated!

File: lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm (hunk 2 above)
Comment (tmooney) [inline lib/perl/Genome/Model/SomaticVariation/Command/SimplifyVcf.pm]: Not likely.  I've changed it to a `warning_message` and cleaned up a bit.

**Comment by ebelter** [comment]
+1

**Comment by acoffman** [comment]
+1
